### ISSUE TITLE
Term2. Sections 11.6 and 11.7

### ISSUE THREAD

**Issue by ricardo-jmnez** [udacity/sdc-issue-reports#1246]
The localization topic is very hard. I really appreciate your efforts in explaining it in a very clear way. I think your walk-through sections are a very good attempt to clarify complex topics. Just make sure they are correct, and take full advantage of your own material.  Consistency helps a lot too! I know this is challenging as many people may be involved in the development of content, perhaps some quality checks may help here. 
 
In this regard, please check the quality of section 11.6 (Calculate Localization Posterior) and 11.7 (Initialize Belief State). Some issues on 11.6 is the consistency of the terminology and correctness of solutions to quiz questions. For instance, you are not consistent with your own terminology as introduced in section 11.5, which I repeat here:

P(location∣observation): Probability of a position given an observation (posterior)
P(observation∣location): Probability of an observation given a position (likelihood)
P(location): Prior probability of a position
P(observation): Total probability of an observation

Furthermore, you introduce new terminology without defining it. What is Normalized P(posterior)? you do not defined it at its introduction, you only give a hint to what it is in the explanation to one solution.  

More importantly, check that the use of Bayes theorem is correct given your own definitions. 

In section 11.7. Please check the first question in the quiz. You contradict yourself. You ask: “What is our initial probability (initial belief state) for position 10?” But the solver assumes you are asking for position 11, I believe. 

Also, please specify the grid spacing! Is it one, two, three meters?

![screen shot 2017-12-04 at 8 59 51 am](https://user-images.githubusercontent.com/26627241/33542737-948a2b38-d8d4-11e7-9ed8-995b1043c044.png)

**Comment by baumanab** [comment]
Thank you for bringing this to our attention.  We have made some changes based on this feedback.  I think all items have been addressed except for specifying the grid spacing.  We are actually working with continuous distance, though we express many of our measurements and tics as integers, so they may appear discrete.  Grid spacing doesn't come into play.  When determining our priors we assume we are at a landmark +/- the position standard deviation.  So, only the value of the standard deviation has an impact.  Does that help?
